docs/plugins/discovery-gce.asciidoc

```diff
@@ -402,13 +425,13 @@ Add `--metadata es_port=9301` option:
 --------------------------------------------------
 # when creating first instance
 gcloud compute instances create myesnode1 \
-       --scopes=compute-rw,storage-full \
-       --metadata es_port=9301
+  --scopes=compute-rw,storage-full \
+  --metadata es_port=9301
 
 # when creating an instance from an image
-gcloud compute instances create myesnode2 --image=elasticsearch-1-0-0-RC1 \
-       --zone europe-west1-a --machine-type f1-micro --scopes=compute-rw \
-       --metadata es_port=9301
+gcloud compute instances create myesnode2 --image=elasticsearch-gce-image \
+  --zone europe-west1-a --machine-type f1-micro --scopes=compute-rw \
```

Comment:
Should this also use --zone=, --machine-type= and --metadata= (rather than space)?